python/lsst/afw/image/testUtils.py

```diff
@@ -370,10 +370,23 @@ def imagesDiffer(image0, image1, skipMask=None, rtol=1.0e-05, atol=1e-08):
     if not np.allclose(valFilledArr1, valFilledArr2, rtol=rtol, atol=atol):
         errArr = np.abs(valFilledArr1 - valFilledArr2)
         maxErr = errArr.max()
-        maxPosInd = np.where(errArr == maxErr)
-        maxPosTuple = (maxPosInd[1][0], maxPosInd[0][0])
-        errStr = f"maxDiff={maxErr} at position {maxPosTuple}; " \
-                 f"value={valFilledArr1[maxPosInd][0]} vs. {valFilledArr2[maxPosInd][0]}"
+        maxAbsInd = np.where(errArr == maxErr)
+        maxAbsTuple = (maxAbsInd[1][0], maxAbsInd[0][0])
+        # NOTE: use the second image, because the numpy test is:
+        # (atol + rtol * absolute(b))
```

Comment:
I trust this is all right, but this comment and the commit message aren't quite enough for me to easily grasp what it is that `allclose` is doing.  Adding `min` and/or `max` to this expression and an inequality might help.